package.json

```diff
@@ -18,6 +18,7 @@
     "eslint-config-next": "13.2.4",
     "firebase": "^9.19.1",
     "material-icons": "^1.13.3",
+    "material-symbols": "^0.5.5",
```

Comment:
For what do we need this? We already have `material-icons`.

Reply:
@PiyushKalyanpy asked for Icons with adjustable Stroke width as the Icons derived from material-symbols don't have provisions for controlling vector stroke width. Therefore, as discussed with @PiyushKalyanpy I implemented this solution Sir.